extensions/link-bundles/src/utils/chrome.ts

```diff
@@ -7,7 +7,7 @@ import { Bundle } from "../types";
 import { decodeUrlSafely } from "./url";
 
 const execAsync = promisify(exec);
-const preferences: Preferences = getPreferenceValues();
```

Comment:
Why did we change this?

its autogenerated in `raycast-env.d.ts`

Reply:
Oh, ESLint was complaining about it so I assumed it was using outdated syntax. Just reverted the change ☺️

Reply:
It should still be the correct one 😊